<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1338</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="10" column="1">
     <widget class="QListWidget" name="listWidget_eventAway"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_ID">
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_Score">
      <property name="text">
       <string>Score</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_HomeTeam">
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QListWidget" name="listWidget_eventHome"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Away:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_Time">
      <property name="text">
       <string>Time:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_GetGameDetails">
      <property name="text">
       <string>Get Game Event</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Score:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Home:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Time:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_GetGameID">
      <property name="text">
       <string>Get Game ID</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_AwayTeam">
      <property name="text">
       <string>ID:</string>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Home Team Event</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Away Team Event</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_FindYoutubeVideo">
      <property name="text">
       <string>FindYoutubeVideo</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_ExportJSON">
      <property name="text">
       <string>Export Full .json</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1338</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QComboBox" name="comboBox_league">
       <item>
        <property name="text">
         <string>italy/serie-a</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>germany/bundesliga</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>spain/laliga</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>england/epl</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>france/ligue-1</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="comboBox_season">
       <item>
        <property name="text">
         <string>2014-2015</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2015-2016</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_LoadTeams">
       <property name="text">
        <string>Load Teams</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_LoadMore">
       <property name="text">
        <string>Load more Games</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_linkResults">
       <property name="text">
        <string>results Link</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_linkTeams">
       <property name="text">
        <string>teams Link</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_launchJS">
       <property name="text">
        <string>LaunchJS</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_LoadGames">
       <property name="text">
        <string>Load Games</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_Teams">
   <property name="windowTitle">
    <string>Teams</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>List</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Home</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Visitor</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QListWidget" name="listWidget_HomeTeams"/>
         </item>
         <item row="1" column="1">
          <widget class="QListWidget" name="listWidget_VisitorsTeams"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Cross</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QTableWidget" name="tableWidget_Season">
           <row>
            <property name="text">
             <string>HomeTeam1</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>HomeTeam2</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>AwayTeam1</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>AwayTeam2</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
